addons/mail/static/src/messaging/component/notification_popover/notification_popover.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.messaging.component.NotificationPopover" owl="1">
+        <div class="o_NotificationPopover">
+            <t t-foreach="notifications" t-as="notification" t-key="notification.localId">
+                <div>
+                    <span name="notification_status" class="o_NotificationPopover_icon">
+                        <t t-if="notification.notification_status === 'sent'">
+                            <i class='fa fa-check' title="Sent" role="img"/>
+                        </t>
+                        <t t-if="notification.notification_status === 'bounce'">
+                            <i class='fa fa-exclamation text-danger' title="Bounced" role="img"/>
+                        </t>
+                        <t t-if="notification.notification_status === 'exception'">
+                            <i class='fa fa-exclamation text-danger' title="Error" role="img"/>
+                        </t>
+                        <t t-if="notification.notification_status === 'ready'">
+                            <i class='fa fa-send-o' title="Ready" role="img"/>
+                        </t>
+                        <t t-if="notification.notification_status === 'canceled'">
+                            <i class='fa fa-trash-o' title="Canceled" role="img"/>
+                        </t>
+                    </span>
+                    <span name="partner_name" class="o_NotificationPopover_name" t-esc="notification.partner.nameOrDisplayName"/>
```

Comment:
also `partner_name` =/= `name`, choose one (same for both, camelCase too)

Reply:
I don't think we use name= when we have a class to target so I'll just remove it. And yes partnerName is better.